sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2Suite.scala

```diff
@@ -219,14 +219,14 @@ class DataSourceV2Suite extends QueryTest with SharedSQLContext {
         assert(spark.read.format(cls.getName).option("path", path).load().collect().isEmpty)
 
         spark.range(10).select('id as 'i, -'id as 'j).write.format(cls.getName)
-          .option("path", path).save()
+          .option("path", path).mode("append").save()
```

Comment:
I don't think this change is needed because the default mode for v2 is append.

Reply:
Before my PR, we write the files twice: once without the save mode, once with append mode.

Now I switch order, to make sure that the second write doesn't specify save mode, and prove the default mode is append.